app/src/main/java/com/mapbox/maps/testapp/examples/MapViewCustomizationActivity.kt

```diff
@@ -19,35 +19,39 @@ class MapViewCustomizationActivity : AppCompatActivity() {
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
+    // Create  a custom CredentialsManager with a token and set it to CredentialsManager.shared, so that all MapViews created with default config will apply this token.
+    CredentialsManager.default.setAccessToken(getString(R.string.mapbox_access_token))
     setContentView(R.layout.activity_map_view_customization)
-    configureMapViewFromXml()
+    // all options provided in xml file - so we just load style
+    mapView.getMapboxMap().loadStyleUri(Style.DARK)
     configureMapViewFromCode()
   }
 
   private fun configureMapViewFromCode() {
-    val mapboxMapOptions = MapboxMapOptions(this).apply {
-      // set initial camera position
-      cameraOptions = CameraOptions.Builder()
-        .center(Point.fromLngLat(-122.4194, 37.7749))
-        .zoom(9.0)
-        .build()
-      // use texture view renderer
-      textureView = true
-      // set other map options
-      mapOptions = MapOptions.Builder()
-        .constrainMode(ConstrainMode.HEIGHT_ONLY)
-        .glyphsRasterizationOptions(
-          GlyphsRasterizationOptions.Builder()
-            .rasterizationMode(GlyphsRasterizationMode.IDEOGRAPHS_RASTERIZED_LOCALLY)
-            .build()
-        )
-        .build()
-      // set token and cache size for this particular map view
-      resourceOptions = ResourceOptions.Builder()
-        .accessToken(getString(R.string.mapbox_access_token))
-        .cacheSize(75_000L)
-        .build()
-    }
+    // set map options
+    val mapOptions = MapOptions.Builder()
+      .constrainMode(ConstrainMode.HEIGHT_ONLY)
+      .glyphsRasterizationOptions(
+        GlyphsRasterizationOptions.Builder()
+          .rasterizationMode(GlyphsRasterizationMode.IDEOGRAPHS_RASTERIZED_LOCALLY)
+          .build()
+      )
+      .build()
+
+    // set token and cache size for this particular map view
+    val resourceOptions = ResourceOptions.Builder()
+      .accessToken(getString(R.string.mapbox_access_token))
+      .cacheSize(75_000L)
+      .build()
+
+    // set initial camera position
+    val initialCameraOptions = CameraOptions.Builder()
+      .center(Point.fromLngLat(-122.4194, 37.7749))
+      .zoom(9.0)
+      .build()
+
+    val mapboxMapOptions =
```

Comment:
mapinitOptions

Reply:
Fixed in https://github.com/mapbox/mapbox-maps-android/pull/287/files